pkg/commands/workload_apply_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"context"
 	"fmt"
 	"os"
+	runtm "runtime"
```

Comment:
why is this import renamed?

Reply:
runtime is already referring to another package `k8s.io/apimachinery/pkg/runtime`
![image](https://user-images.githubusercontent.com/108013945/191988768-1d652902-1e80-4734-8381-62e1815abb49.png)